Documentation/ApiOverview/GlobalValues/Typo3ConfVars/BE.rst

```diff
@@ -653,6 +653,20 @@ $GLOBALS['TYPO3_CONF_VARS']['BE']['defaultUserTSconfig']
 
    Contains the default user TSconfig.
 
+   Never set this configuration variable directly. Use the API method instead:
```

Comment:
This is fine but the word "never" feels like a dare to me, and makes me want to do it. Is there a reason we can add? "Never set this configuration variable directly because the world will explode. Use the API method instead:"

Reply:
site admins can decide to extend that string, when they want to, for whatever reason. the api method doesn't do it much differently and only adds '[GLOBAL]' in front to avoid side effects of eventual previous conditions. casual extensions like a ext:news would of course always use the API method.